ops/testing.py

```diff
@@ -1063,6 +1063,18 @@ def autostart_services(self, timeout: float = 30.0, delay: float = 0.1) -> pebbl
             if startup == pebble.ServiceStartup.ENABLED:
                 self._service_status[name] = pebble.ServiceStatus.ACTIVE
 
+    def replan_services(self, timeout: float = 30.0, delay: float = 0.1) -> pebble.ChangeID:
+        for name, service in self._render_services().items():
+            # TODO: jam 2021-04-20 This feels awkward that Service.startup might be a string or
+            #  might be an enum. Probably should make Service.startup a property rather than an
+            #  attribute.
+            if service.startup == '':
+                startup = pebble.ServiceStartup.DISABLED
+            else:
+                startup = pebble.ServiceStartup(service.startup)
```

Comment:
Should we make a little helper method for these lines? They're used three times now. It would also DRY up the comment.

Reply:
I think this whole thing could probably undergo a rewrite, but maybe for another time.